### ISSUE TITLE
[JOSS] Dependency ncplot is not well documented

### ISSUE THREAD

**Issue by platipodium** [pmlmodelling/nctoolkit#66]
The dependency `ncplot` is not very well documented.  Any suggestions  or plans to improve that?

https://ncplot.readthedocs.io/en/latest/index.html

See https://github.com/openjournals/joss-reviews/issues/5494

**Comment by robertjwilson** [comment]
I've just checked this. For some reason readthedocs was disliking a notebook. I've replaced the .ipynb file with a .rst file.

https://ncplot.readthedocs.io/en/latest/index.html

**Comment by platipodium** [comment]
OK, I see.  There is indeed only *one* API call: `view`.  That simple :=).  I guess I was missing information on how to style the plot once it is done with `view`.  That should be documented

- what kind of plot object is created?
- with what API is the plot object to be manipulated?

That's at least what I was wondering when I played around with using nctoolkit to viz some of my own files.

**Comment by platipodium** [comment]
I think your comment in the docs 

> This is a package that aims to deliver plotting for rapid exploratory analysis, and therefore it does not offer a large number of customizations. However, because it is built on hvplot, you can use most of the customization options available in hvplot, which are detailed [here](https://hvplot.holoviz.org/user_guide/Customization.html). Arguments such as logz and clim can be passed to plot and will be automatically passed to the hvplot method used.

is exactly what I was looking for.   Thanks!

Now, however, the readthedocs is broken.  I'll create a new issue
